server/controllers/user.js

```diff
@@ -11,9 +49,14 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
   if (searchString) {
     users = await User.find({
       username: { $regex: searchString, $options: "i" },
-    });
+    })
+      .select(" -password  ")
+      .populate("profile", "firstName lastName")
+      .exec()
+      .then((result) => {
```

Comment:
We can get rid of `then` and `await` since it's we're using an `async` function.